src/ceph-volume-zfs/bin/ceph-volume-zfs

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
+
+from ceph_volume_zfs import main
+if __name__ == '__main__' and __package__ is None:
```

Comment:
why is this section needed?

Reply:
@alfredodeza 
2 possibilities, and I'm not sure which one:
1) It is needed, so I can also call this as a standalone program: ceph-volume-zfs
2) It was generated by the tool I used to generate the module framework to begin with.

But if you think it can/should go, I'll test that.